Readme modification (#1700)
huggingface/optimum-habana: examples/text-classification/README.md

GLUE is made up of a total of 9 different tasks where the task name can be cola, sst2, mrpc, stsb, qqp, mnli, qnli, rte or wnli.

> Finetune model can be done as single device, multi-devices with mpi or deepspeed.

Reviewer: some re wording: Fine-tuning of the model can be performed on a single device, across multiple devices using MPI, or with DeepSpeed.

> For multi-devices training with mpi, add the following before run_glue.py where X is device number 

Reviewer: mpi -> MPI

> For multi-devices training with MPI, add the following before run_glue.py. X is device number 

Reviewer 2: ```suggestion
For multi-device training with MPI, add the following before `run_glue.py` (X is the device number):
```

> python ../gaudi_spawn.py \

> >    --world_size X --use_mpi run_glue.py \

Reviewer 2: Let's rather put the code snippet between ``` and ``` than using >

> For multi-devices training with deepspeed, add the following before run_glue.py and deepspeed config with --deepspeed after where X is device number

Reviewer: deepspeed -> DeepSpeed

> For multi-devices training with DeepSpeed, add the following before run_glue.py and deepspeed config with --deepspeed after. X is device number

Reviewer 2: ```suggestion
For multi-device training with DeepSpeed, add the following before `run_glue.py` and DeepSpeed config with `--deepspeed` after (X is the device number):
```

> Finetune model can be done as single device, multi-devices with mpi or deepspeed.
> 
> For multi-devices training with mpi, add the following before run_glue.py where X is device number 
> > python ../gaudi_spawn.py \
> >    --world_size X --use_mpi run_glue.py \
> 
> For multi-devices training with deepspeed, add the following before run_glue.py and deepspeed config with --deepspeed after where X is device number
> > python ../gaudi_spawn.py \
> >    --world_size X --use_deepspeed run_glue.py --deepspeed \
> 

Reviewer 3: ```suggestion
Fine-tuning a model can be done on a single device or across multiple devices using MPI or DeepSpeed.

For multi-device training with MPI, add the following before run_glue.py, where X is the device number:
> python ../gaudi_spawn.py \
>    --world_size X --use_mpi run_glue.py \

For multi-device training with DeepSpeed, add the following before run_glue.py and include the DeepSpeed configuration with --deepspeed after, where X is the device number:
> python ../gaudi_spawn.py \
>    --world_size X --use_deepspeed run_glue.py --deepspeed \

```

Minor rewording. Please disregard if you do not agree.

> >    --world_size X --use_deepspeed run_glue.py --deepspeed \

Reviewer 2: ```suggestion
>    --world_size X --use_deepspeed run_glue.py --deepspeed path_to_deepspeed_config \
```

> >    --world_size X --use_deepspeed run_glue.py --deepspeed path_to_deepspeed_config \

Reviewer 2: Same

> If your model classification head dimensions do not fit the number of labels in the dataset, you can specify `--ignore_mismatched_sizes` to adapt it.

## Fine-tuning Llama Guard on MRPC

Llama Guard can be used for text classification. The Transformers library will change the head of the model for you during fine-tuning or inference. You can use the same general command as for BERT, except you need to add `--add_pad_token=True` because Llama based models don't have a `pad_token` in their model and tokenizer configuration files. So `--add_pad_token=True` will add a `pad_token` equal to the `eos_token` to the tokenizer and model configurations if it's not defined.

> Here is the example of Llama Gaurd text classification MRPC task finetune with DeepSpeed on 8 devices:

Reviewer 2: ```suggestion
Here is the example of Llama Guard text classification MRPC task finetuning with DeepSpeed on 8 devices:
```
